cli

```diff
@@ -19,6 +20,10 @@ try:
     tickers = fetch(session, args.limit)
     stocks = [scrape(session, ticker) for ticker in tickers]
 
+    for stock in stocks:
+        calc_params(stock)
```

Comment:
I don't think analyser.py should export anything apart from the main analyser function
I think it's implicit that calling the analyser would calculate these ratios?
```suggestion
       analyse(stock)
```